financial-connections/src/main/java/com/stripe/android/financialconnections/features/linkaccountpicker/LinkAccountPickerViewModel.kt

```diff
@@ -194,35 +195,55 @@ internal class LinkAccountPickerViewModel @AssistedInject constructor(
         suspend {
             val state = stateFlow.value
             val payload = requireNotNull(state.payload())
-
             val accounts = payload.selectedAccounts.map { it.account }
+            val selectedAccountDrawers = payload.selectedAccounts
```

Comment:
Nit: Maybe we can name this `drawersToPresentOnSubmission` to make it extra clear.

Reply:
adding this on a PR with small changes coming today!